src/components/Nav/index.tsx

```diff
@@ -92,16 +94,37 @@ const Nav: FC<IProps> = ({ path }) => {
                   variant="ghost"
                   isSecondary
                   px={1.5}
+                  _hover={{
+                    transform: "rotate(30deg)", // Rotate the icon to 30 degrees on hover
+                    color: "primary.hover", // Change color to blue on hover
+                  }}
                   onClick={toggleColorMode}
                 ></IconButton>
+
                 <ButtonLink
+                  onMouseOver={() => setLanguagesHover(true)}
+                  onMouseOut={() => setLanguagesHover(false)}
                   to={`/languages/${fromPageParameter}`}
-                  leftIcon={<Icon as={MdLanguage} />}
                   variant="ghost"
                   isSecondary
                   px={1.5}
+                  _hover={{
+                    color: "primary.hover", // Change color on hover
+                  }}
                 >
-                  {t("languages")} {i18n.language.toUpperCase()}
+                  <Box mr={2} mt={2}>
+                    {" "}
+                    {/* Add spacing between the text and the icon */}
+                    <Icon
+                      as={MdLanguage}
+                      transition="transform 0.3s ease-in-out, color 0.2s"
+                      // _groupHover renders "[data-group]:hover &"
```

Comment:
✂️

Reply:
I didn't get this😅

Reply:
Sorry. I meant to cut it, to remove it xD

Its a comment that doesn't add any new information to the developer.